src/game/game.gd

```diff
@@ -0,0 +1,20 @@
+extends Node2D
+
+onready var mapNode: Node2D = $Map
+onready var characterNode: Node2D = $Characters
+onready var rng = RandomNumberGenerator.new()
```

Comment:
A very small nitpick, but we seem to have a style going where we type all of our variables, so maybe type this one to be RandomNumberGenerator